Firmware/ultralcd.cpp

```diff
@@ -8972,6 +8972,15 @@ bool FarmOrUserECool(){
     return farm_mode || UserECoolEnabled();
 }
 
+#ifdef PRUSA_SN_SUPPORT
+void WorkaroundPrusaSN() {
+    const char *SN = PSTR("CZPX000000000000000");
```

Comment:
We decided not to assign a valid serial number to such printers, instead let's assign something human-readable:
`CZPX000000000000000`
`CZPXInvalidSerialNr`

Reply:
done